discojs/src/tasks/simple_face.ts

```diff
@@ -39,48 +39,12 @@ export const task: Task = {
   }
 }
 
-export function model (imageHeight = 200, imageWidth = 200, imageChannels = 3, numOutputClasses = 2): tf.LayersModel {
-  const model = tf.sequential()
-
-  // In the first layer of our convolutional neural network we have
-  // to specify the input shape. Then we specify some parameters for
-  // the convolution operation that takes place in this layer.
-  model.add(tf.layers.conv2d({
-    inputShape: [imageHeight, imageWidth, imageChannels],
-    kernelSize: 5,
-    filters: 8,
-    strides: 1,
-    activation: 'relu',
-    kernelInitializer: 'varianceScaling'
-  }))
-
-  // The MaxPooling layer acts as a sort of downsampling using max values
-  // in a region instead of averaging.
-  model.add(tf.layers.maxPooling2d({ poolSize: [2, 2], strides: [2, 2] }))
-
-  // Repeat another conv2d + maxPooling stack.
-  // Note that we have more filters in the convolution.
-  model.add(tf.layers.conv2d({
-    kernelSize: 5,
-    filters: 16,
-    strides: 1,
-    activation: 'relu',
-    kernelInitializer: 'varianceScaling'
-  }))
-  model.add(tf.layers.maxPooling2d({ poolSize: [2, 2], strides: [2, 2] }))
-
-  // Now we flatten the output from the 2D filters into a 1D vector to prepare
-  // it for input into our last layer. This is common practice when feeding
-  // higher dimensional data to a final classification output layer.
-  model.add(tf.layers.flatten())
-
-  // Our last layer is a dense layer which has 10 output units, one for each
-  // output class (i.e. 0, 1, 2, 3, 4, 5, 6, 7, 8, 9).
-  model.add(tf.layers.dense({
-    units: numOutputClasses,
-    kernelInitializer: 'varianceScaling',
-    activation: 'softmax'
-  }))
+export async function model (): Promise<tf.LayersModel> {
+  // TFJSNODE is define in the server, so will only be imported there
+  if (typeof process.env.TFJSNODE === 'string') {
```

Comment:
nice 🚀 but the function would cause trouble if called within another env than node, i.e. somewhere the `process` wouldn't be defined

Reply:
changed it to check if window is define (since it is the only case we are outside of nodejs)